Automattic/wp-calypso
# Add tests for new mailboxes unified forms

#### Changes proposed in this Pull Request

This changes are intended to add new automated tests for the [new mailbox form unification](https://github.com/Automattic/wp-calypso/pull/63038)

#### Testing instructions

1. Check out this branch locally git checkout `test/hydrate-mailboxes-and-fields-api`
2. Open a terminal
3. Type: `yarn test-client "client/my-sites/email/form/mailboxes/test/validator.ts"`

```
Expected output:
Test Suites: 1 passed, 1 total
Tests:       6 passed, 6 total
Snapshots:   0 total
```


Related to https://github.com/Automattic/wp-calypso/pull/63038
